.buildkite/hooks/pre-command

```diff
@@ -13,7 +13,8 @@ export PS4="++"
 #
 eval "$(ci/channel-info.sh)"
 eval "$(ci/sbf-tools-info.sh)"
-export CARGO_TARGET_CACHE=$HOME/cargo-target-cache/"$CHANNEL"-"$BUILDKITE_LABEL"-"$SBF_TOOLS_VERSION"
+source "ci/rust-version.sh"
+export CARGO_TARGET_CACHE=$HOME/cargo-target-cache/"$CHANNEL"-"$BUILDKITE_LABEL"-"$SBF_TOOLS_VERSION"-"$rust_stable"
```

Comment:
I think we've seen the cache corrupted when bumping nightlies too (happens sometimes when bugs). Might not be worth the trouble ATM though.  I wonder if we should chop of the patch version component? 

We should also consider switching from channel to base branch to avoid unnecessarily purging caches on beta -> stable advancements

Reply:
I chopped the patch version component.